content/id/docs/tasks/job/automated-tasks-with-cron-jobs.md

```diff
@@ -0,0 +1,213 @@
+
+Seperti semua konfigurasi Kubernetes, sebuah CronJob membutuhkan bidang `apiVersion`, `kind`, dan `metadata`. Untuk informasi
+umum tentang bekerja dengan berkas konfigurasi, lihat dokumentasi [men-deploy aplikasi](/docs/user-guide/deploying-applications),
+dan [mengunakan kubectl untuk manajemen sumber daya](/docs/user-guide/working-with-resources).
+
+Sebuah konfigurasi CronJob juga membutuhkan sebuah [bagian `.spec`](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status).
+
+{{< note >}}
+Semua modifikasi pada sebuah CronJob, terutama `.spec`, akan diterapkan pada proses berikut.
+{{< /note >}}
+
+### Penjadwalan
+
+`.spec.schedule` adalah bidang yang wajib diisi dari sebuah `.spec`
+Dibutuhkan sebuah fromat string [Cron](https://en.wikipedia.org/wiki/Cron), misalnya `0 * * * *` atau `@hourly`, sebagai jadwal Job untuk dibuat dan dieksekusi.
+
+Format ini juga mencakup nilai langkah `Vixie cron`. Seperti penjelasan di [FreeBSD manual](https://www.freebsd.org/cgi/man.cgi?crontab%285%29):
+
+> Nilai langkah dapat digunakan bersama dengan rentang. Sebuah rentang diikuti dengan
+> `/<number>` menentukan lompatan angka melalui rentang. 
+> Misalnya, `0-23/2` dapat digunakan dalam jam untuk menentukan
+> perintah akan dieksekusi setiap jam (alternatif dalam bawaan v7 adalah
+> `0,2,4,6,8,10,12,14,16,18,20,22`). Langkah-langkah juga diizinkan setelah
+> tanda bintang, jadi jika kamu menginginkan "setiap dua jam", gunakan saja `*/2`.
+
+{{< note >}}
+Sebuah tanda tanya (`?`) dalam penjadwalan memiliki makna yang sama dengan tanda bintang `*`, yaitu, singkatan dari nilai apa pun yang tersedia untuk bidang tertentu.
+{{< /note >}}
+
+### Templat Job
+
+`.spec.JobTemplate` adalah templat untuk sebuah Job, dan itu wajib.
+Templat Job memiliki skema yang sama dengan [Job](/docs/concepts/workloads/controllers/jobs-run-to-completion/), kecuali jika bersarang dan tidak memiliki sebuah `apiVersion` atau `kind`.
+Untuk informasi lebih lanjut tentang menulis sebuah Job `.spec` lihat [Menulis spesifikasi Job](/docs/concepts/workloads/controllers/jobs-run-to-completion/#writing-a-job-spec).
+
+### _Starting Deadline_
+
+Bidang `.spec.startingDeadlineSeconds` adalah bidang opsional.
+Bidang tersebut berarti batas waktu dalam satuan detik untuk memulai sebuah Job jika Job melewatkan waktu yang telah dijadwalkan karena alasan apapun.
+Setelah mencapai batas waktu, CronJob tidak akan memulai sebuah Job.
+Job yang tidak memenuhi batas waktu, dengan cara ini dianggap sebagai Job yang gagal.
+Jika bidang ini tidak ditentukan, maka Job tidak memiliki batas waktu.
+
+CronJob Controller menghitung berapa banyak jadwal yang terlewat untuk sebuah CronJob. jika lebih dari 100 jadwal yang terlewat, maka tidak ada lagi CronJob yang akan dijadwalkan. Ketika `.spec.startingDeadlineSeconds` tidak disetel, CronJob Controller menghitung jadwal yang terlewat dari `status.lastScheduleTime` hingga sekarang.
```

Comment:
is Controller an API resource? if not I think we should italicize the word instead, same comment applies for the rest use of "CronJob Controller" in this page
```suggestion
_Controller_ CronJob menghitung berapa banyak jadwal yang terlewat untuk sebuah CronJob. jika lebih dari 100 jadwal yang terlewat, maka tidak ada lagi CronJob yang akan dijadwalkan. Ketika `.spec.startingDeadlineSeconds` tidak disetel, CronJob Controller menghitung jadwal yang terlewat dari `status.lastScheduleTime` hingga sekarang.
```